### DISCUSSION TITLE
Issue with version on fdroid vs playstore

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#2056] [General Discussion]
This app when installed from fdroid or github page (latest version) still shows an update from google play store. 
If we click on update, it shows error.

I have observed in certain apps such as telegram that when we download it from its webpage and install it (direct download without using play store) play store is unable to recognise that app is already installed and still shows "install" option and we can download and install a separate play store version of the same app. 

I think it will be good if this option is provided in this app also so that when we download it from non-play store options such as fdroid or github page, those versions are unreadable to google play store.

This feature will be particularly useful to those who do not like downloading from google play store.

Thanks

**Comment by KyleSCraig** [comment]
I suggest adding this perspective to issue #2000

**Comment by Samiimasmoudii** [comment]
Maybe because apps installed from non-Play Store sources often have the same package name as their Play Store counterparts? and Android’s system identifies them as the same app.

**Reply by ghost** [reply]
I am not a techie but i have observed in certain apps e,g; Telegram, if you download and install it directly from Telegram webpage using direct download link (non play store), you can still install it from play store again (it is just like you have not already installed from any source). Play store fails to recognise it is already installed in phone. Also this way it gets separate updates which don't use play store but we get notification in telegram that a new update is available.
This app being offline will fail to show update notification in app if it implements the above Telegram like feature but i think we can take that little pain to check fdroid or github manually for updates.
I have requested this feature for those who like fdroid and don't want play store bothering them with update notification which won't go away till you uninstall the fdroid version and install the play store version of the app.

**Reply by Samiimasmoudii** [reply]
Implementing a similar feature in this app could definitely benefit users who prefer non-Play Store options like F-Droid or GitHub.

While I understand that the app may not notify users about updates, the trade-off might be worthwhile for those looking to avoid Play Store notifications. It could be better for users who prefer manual checks for updates. Hopefully, this feature can be considered for future releases!
